README.md

```diff
@@ -175,6 +185,68 @@ jobs:
           body: ${{ steps.calibre.outputs.markdown }}
 ```
 
+## Combined workflow
+
+You can combine all of the above into one all-encompassing workflow
+
+```yml
+# Compress images:
+# - on pull requests from this report containing images

```

Comment:
Could you clarify this line? I'm not quite sure what it's saying

Reply:
Changed it to this:

```yml
# image-actions will run in the following scenarios:
# - on pull requests containing images (not including forks)
# - on pushing of images to master (for forks)
```

That clearer?